### PR TITLE
Update camshaft reference to 0.33.2

### PR DESCRIPTION
**Pull request by ethervoid**
In order to fix a problem with postal geocoding analysis

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "bootstrap-colorpicker": "2.5.0",
     "browserify": "13.0.0",
     "browserify-shim": "3.8.12",
-    "camshaft-reference": "0.33.0",
+    "camshaft-reference": "0.33.2",
```
Comment (nobuti) [inline package.json]: what did happen to 0.33.1?

Comment (ethervoid) [reply package.json]: I made two changes and it automatically increased the version and I didn't notice and I published it 😓

**Comment by ethervoid** [comment]
@nobuti check again pls

**Comment by ethervoid** [comment]
@nobuti  Tested in staging and working properly :)

**Comment by nobuti** [comment]
Do you feel confident to deploy it today or you prefer to wait until Monday? 💪  or 🐔 ?

**Comment by ethervoid** [comment]
I'll do it monday xD

**Pull request closed without merging**